AssemblyDiffer/Providers/NuGet/NuGetPackage.cs

```diff
@@ -10,6 +10,10 @@ namespace Differ.Providers.NuGet
 {
 	public class NuGetPackage : NuGetPackageBase
 	{
+		public static NuGetPackage Skip { get; } = new NuGetPackage();
```

Comment:
Would `None` or `Null` be more descriptive?

Reply:
Went with `NotFound`, naming things.. :)